instance/management/commands/activity_csv.py

```diff
@@ -108,10 +109,11 @@ def handle(self, *args, **options):  # pylint: disable=too-many-locals
                     self.stderr.write(self.style.SUCCESS(line))  # pylint: disable=no-member
                 process.wait()
 
-            csv_writer = csv.writer(out)
-            csv_writer.writerow(
-                ['Appserver IP', 'Owner Emails', 'Unique Hits', 'Total Users', 'Total Courses', 'Age (Days)']
-            )
+            csv_writer = csv.writer(out, quoting=csv.QUOTE_NONNUMERIC)
+            csv_writer.writerow([
+                'Appserver IP', 'Internal LMS Domain', 'Name', 'Contact Email', 'Unique Hits', 'Total Users',
+                'Total Courses', 'Age (Days)'
+            ])
```

Comment:
@pomegranited True - although for this time we don't need to be too precise, activity within the lifespan of the last provisioned appserver seemed enough to get a rough idea.

Eventually, now that event logs are saved for all instances, we will be able to go get even more accurate activity stats there; but this was a too recent change to make it useful here.